spec/10-expressions.md

```diff
@@ -2930,7 +2930,9 @@ At least one of the elements of the *list-expression-list* must be non-empty.
 **Semantics**
 
 This intrinsic assigns one or more elements of the source array to the
-target variables. On success, it returns a copy of the source array. If the
+target variables. On success, the target variable will be assigned to the
```

Comment:
@bp1222 This change removes the information what `list()´ returns on success.

Reply:
I tried to keep it clear in the second sentence that it will either return a copy of the value, or a reference to the actual value.

Reply:
I'm not referring to the individual values (this part looks fine), but rather to the result of `list()`, which is still a copy of the source array. E.g. after`$res = list(&$one, $two) = $a;`, `$res` is a copy of `$a`.